vk-video/src/parser/au_splitter.rs

```diff
@@ -152,3 +153,6 @@ fn idrs_where_idr_pic_id_differs(last: &Slice, curr: &Slice) -> bool {
         _ => false,
     }
 }
+
+/// Group of [`Nalu`]s representing one frame
+pub struct AccessUnit(pub Vec<Nalu>);
```

Comment:
If it's a finished AU, the storage won't grow. Maybe make it into a boxed slice to communicate that?